### ISSUE TITLE
BREAK and END keyword replacing other texts

### ISSUE THREAD

**Issue by xitizmandal** [lynchjames/obsidian-day-planner#40]
Typing any word that starts with `break` converts the whole text into `BREAK`. Similar is the case with `end`.

Obsidian version: 0.9.17
Day Planner version: [0.5.3](https://github.com/lynchjames/obsidian-day-planner/releases/tag/0.5.3)

Eg:

```
- [ ] 08:30 breakout sessions
```
This is converted to

```
- [ ] 08:30 BREAK
```

In case of `end...`

```
- [ ] 08:30 endgame
```

This is converted to

```
- [ ] 08:30 END
```

**Comment by lynchjames** [comment]
🤦 regex should be checking for end of line or at least a space for these special words. I'll get a fix in for this in the next release.

**Comment by lynchjames** [comment]
This should now be fixed in the latest release ([0.5.4](https://github.com/lynchjames/obsidian-day-planner/releases/tag/0.5.4)).
